test/integration/test-video-players.js

```diff
@@ -28,9 +28,8 @@
 
 import {runVideoPlayerIntegrationTests} from './test-video-players-helper';
 
-//TODO(aghassemi,#7822): We have to skip iOS for video tests since videos
-// can't play SauceLab's iOS simulator. We need real devices instead.
-
+// TODO(aghassemi,#7822): We have to skip iOS for video tests since videos
+// can't play on an iOS simulator. We need real devices instead.
```

Comment:
Same as above, what do you think about removing or replacing this comment block until we know? In particular "videos can't play on an iOS simulator" doesn't seem particularly accurate since it is making a sweeping statement. Perhaps something like `"Videos may not be playable on an iOS simulator. See #7822"` if, when reintroducing tests, it may be helpful to leave some track record.

Reply:
Agree. Done.